src/storage/memory.ts

```diff
@@ -67,6 +69,16 @@ export function buildMemoryStorage(
         }
       }
 
+      if (cloneData === 'double' && value !== undefined) {
```

Comment:
```suggestion
      /* istanbul ignore if 'only available on super recent browsers' */
      if (cloneData === 'double' && value !== undefined) {
```

Add this comment to code coverage skip the structuredClone test